### DISCUSSION TITLE
Pensar em ferramentas de observabilidade

### DISCUSSION THREAD

**Discussion by andrewarosario** [techexpertspro/portal-ascepa#9] [Ideas]
Ferramentas de observabilidade fornecem insights acionáveis sobre o desempenho, a saúde e o comportamento das aplicações front-end, permitindo que desenvolvedores e equipes de operações identifiquem e resolvam problemas de forma proativa.

**Comment by jpsthome** [comment]
Uma possível solução é a implementação de algum bot no servidor do Discord que envie mensagens com os erros, abri uma [discussão](https://github.com/techexpertspro/portal-ascepa/discussions/11) sobre esse assunto em específico.
